mindmeld/components/classifier.py

```diff
@@ -454,6 +450,16 @@ def dump(self, model_path, incremental_model_path=None):
             if path == model_path:
                 self.dirty = False
 
+    def _dump(self, path):
```

Comment:
maybe we can make this into an abstract method?

Reply:
@vijay120 not all classifiers have dump-able data, so i think making it into abstract method might not be necessary and the default action of the method being not to dump anything.